config/auth.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Defaults
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default authentication "guard" and password
+    | reset options for your application. You may change these defaults
+    | as required, but they're a perfect start for most applications.
+    |
+    */
+
+    'defaults' => [
```

Comment:
My app (completed in a private instance of Github), currently works without this configuration file. If your update to Lumen module requires this config file, it is backwards incompatible change and it shouldn't be released in a patch release.